pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -78,7 +94,7 @@ func (pc *PackageControllerClient) InstallController(ctx context.Context) error
 	return nil
 }
 
-func (pc *PackageControllerClient) ValidateControllerDoesNotExist(ctx context.Context) error {
+func (pc *PackageControllerClient) IsInstalled(ctx context.Context) error {
```

Comment:
Based on the name, should't this return a boolean?
Like:
```suggestion
func (pc *PackageControllerClient) IsInstalled(ctx context.Context) (bool, error) {
```